### DISCUSSION TITLE
Monster Data Loading

### DISCUSSION THREAD

**Discussion by DakkJaniels** [diasurgical/DevilutionX#4148] [Ideas]
Related to #4147 and #4145 would it make sense to stop loading a bunch of stuff from the save file, if it's all just the standard table data? I can understand the stuff that changes (e.g., position, hit points, etc.) but why load things like AC or resists from the save file when they can be just grabbed from the data table, especially if we are going to go through the process of associating the standard info anyway when MData is assigned to the monster in SyncMonsterAnim()? Of course, for compatibility purposes, we would still have to save things correctly, but if we disregard the file on loading for the stats that are static, we don't have to be as concerned with changing things so that they still fit within the save file.  Thoughts?

**Comment by AJenbo** [comment]
Sounds good
